csrc/host_ir/host_ir.h

```diff
@@ -115,6 +115,35 @@ class PostOnStream : public Expr {
   }
 };
 
+class LaunchKernel : public Expr {
+ public:
+  using Expr::Expr;
+  LaunchKernel(
+      IrBuilderPasskey passkey,
+      int64_t hic_executor_index, // Index into the HostIrContainer's vector of
+                                  // KernelExecutors--i.e., the kernel this IR
+                                  // should launch
+      std::vector<Val*> inputs,
+      std::vector<Val*> outputs);
```

Comment:
```suggestion
      const std::vector<Val*>& inputs,
      const std::vector<Val*>& outputs);
```
so it's easier to avoid unnecessary copies -- no std::move needed.